### DISCUSSION TITLE
Problems with encoding/decoding JXL

### DISCUSSION THREAD

**Discussion by Dattel** [dlemstra/Magick.NET#1228] [Help]
### Magick.NET version

Current

### Environment (Operating system, version and so on)

Win10, VS2022, Magick.NET-Q8-AnyCPU

### Description

Hi there,
are there any known issuses regarding JXL? Im trying to load a JXL image with the latest version and it takes about 6s to load these image using the constructor. 

I figured out that if i downgrade Magick.NET-Q8-AnyCPU and  Magick.NET.Core to v9.1.2, the same image takes about 200ms to load. 

Also i'm not able to load a jpg and write it back to disk as jxl. The write function just freezes and don't came back.

Thanks for your help.

### Steps to Reproduce

var i = new MagickImage(pathToJXLFile); -> took 6s

var j = new MagickImage(pathToJPG);
j.Write("C:\\Temp\\image1.jxl",MagickFormat.Jxl);  -> freezes without exception

**Comment by dlemstra** [comment]
The encoder of jxl is not that fast so you probably will need to wait a bit longer. And on the @ImageMagick side we had to disable code optimizations due to an issue with VisualStudi deadlocking when building the current code. That is probably causing this slowdown. When the next version of libjxl is released we can check if we can enable the compiler optimizations again.

**Reply by Dattel** [reply]
Thanks for your quick reply,
i read of these situation with compiler optimizations at another issue.

So i probably stuck in these deadlock because i hardly thinky, that transcoding one JXL with it's larges size of 2500px should take more than 5 minutes of waiting.. :-(

Update: okay, one image took indead more than 5 minutes to encode...
